internal/github/profile_test.go

```
@@ -0,0 +1,330 @@
+
+	validatedProfile, err := github.ValidateProfile(context.Background(), profile)
+	require.NoError(t, err)
+	// Test that we get an error attempting to load it before fetching
+	_, err = profileStore.GetOrganization()
+	require.Error(t, err)
+
+	orgProfile, err := github.FetchOrganizationProfile(configURL, gh)
+	require.NoError(t, err)
+	assert.Equal(t, validatedProfile, orgProfile)
+
+	orgProfile, err = github.FetchOrganizationProfile(configURL, gh)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)